cloud/src/meta-service/meta_service.cpp

```diff
@@ -1981,6 +2002,7 @@ void MetaServiceImpl::update_delete_bitmap(google::protobuf::RpcController* cont
         LOG(WARNING) << msg << ", cloud_unique_id=" << request->cloud_unique_id();
         return;
     }
+    check_version(use_version, instance_id);
```

Comment:
`check_version` parses the config string every time. Will there be any performance issues with this?

Reply:
If don't parses every time, it can make config update immediately work